src/Classes/Executor.php

```diff
@@ -40,8 +40,8 @@ abstract class Executor
     /**
      * Executor constructor.
      *
-     * @param  NotifierFactory|null  $notifierFactory
-     * @param  Client|null  $httpClient
+     * @param NotifierFactory|null $notifierFactory
```

Comment:
It seems like there's still a lot of comments and docblocks that have been updated. Please could you revert all these back to how they were?